### ISSUE TITLE
Debugger not working in WSL

### ISSUE THREAD

**Issue by RapidMovies** [microsoft/vscode-makefile-tools#228]
I was just testing out Makefile Tools on WSL (opened the folder via the "code" command of the "Remote - WSL" extension), and everything in the extension works fine, except for the debugger. I just have 4 simple files: https://github.com/RapidMovies/MakefileToolsTesting/tree/main.

So here's everything I did:
1) Set the configuration provider to Makefile Tools
2) Set the build and launch targets 
![image](https://user-images.githubusercontent.com/94209242/141603734-2d6086c8-eceb-447d-8012-ee405acd1258.png)
3) Set the -h argument in my settings.json file. ("test" is the directory with all the files)
![image](https://user-images.githubusercontent.com/94209242/141603782-40535c2f-e35f-47d9-ada6-cdca7aa0ccfd.png)
4) Build the project
![image](https://user-images.githubusercontent.com/94209242/141603827-593393e4-0b88-4e51-bf24-9a6afdaa7e3b.png)
5) Set a breaking point in main.c for testing purposes.
![image](https://user-images.githubusercontent.com/94209242/141603850-589b0ead-9898-4c8c-9fe9-0350a42ea14f.png)
6) Hit debug 
![image](https://user-images.githubusercontent.com/94209242/141604084-57937d54-9e37-40d2-aed3-20e485c452a4.png)

I'm not sure what this debug console message means, but I at least know it's not working as intended. Is this is a bug in the current build of Makefile Tools, or am I actually doing something wrong?

**Comment by andreeis** [comment]
@RapidMovies, from what I see everything that you do is correct. I didn't get a chance to test yet this extension on WSL. It was on my to-do list for the future.
Until I get to create a WSL environment on my side, @WardenGnaw does this ring any bells?

**Comment by Yorkzero** [comment]
@RapidMovies, hi, bro. I'm new to GCC and Makefile, so I had a similar problem not too long ago. Maybe you could try adding the arg **_“-g”_**  to your Makefile like following, which way help me to debug target sucessfully:
```
main: main.c
	gcc -o main -g main.c
clean:
	rm main
	rm *.o
```
BTW, I was using _remote-ssh_ to connect my ubuntu server to do all these things, so this way is probably similar to the WSL? I'm not sure. Anyway, this plugin is very helpful for my study!

**Comment by RapidMovies** [comment]
@Yorkzero I added the -g flag to all the Makefile commands. It was still showing the same errors at first, _until_ I removed the -h argument from setting.json. That seemed to have done the trick! Now it's working semi-consistently.

@andreeis I got it to work semi-consistently, but I'm still running into some other bugs _occasionally._ For example, if I add a breakpoint to the "print1to10" function inside of main.c, then try to step into that function, it'll say that the print1to10 address couldn't be located (since it's in a different file I assume). I don't actually run into this every single time, which is strange. I can make a separate post about it, as soon as I figure out how to recreate this bug consistently.
